jupyterlab_bigquery/cloudbuild-release.yaml

```diff
@@ -0,0 +1,33 @@
+substitutions:
```

Comment:
This can be removed as we won't be using Cloud Build to release these extensions.